scripts/lint-packages.sh

```diff
@@ -119,6 +119,28 @@ check_indentation() {
 	return 0
 }
 
+# Check the latest commit that modified `$package`
+# It must either:
+# - Modify TERMUX_PKG_REVISION
+# - Modify TERMUX_PKG_VERSION
+# - Or specify one of the CI skip tags
+check_version_change() {
+	local base_commit commit_diff package="$1"
+	base_commit="$(git merge-base --fork-point origin/master)"
+	commit_diff="$(git log --patch "${base_commit}.." -- "$package")"
+
+	# If the diff is empty there's no commit modifying that package on this branch, which is a PASS.
+	[[ -z "$commit_diff" ]] && return
+
+	grep -q \
+		-e '^+TERMUX_PKG_REVISION=' \
+		-e '^+TERMUX_PKG_VERSION=' \
+		-e '\[no ci\]' \
```

Comment:
It is better to use a tag to skip this check, such as `[no check version change]`.

Sometimes the auto-update bot will commit some changes on which the CI doesn't pass. Currently the bot only checks the build for aarch64. If other arches fail, no revision change is actually needed, thus this check should be skipped.